src/ConfigurationContext.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, createContext, useContext } from "react"
+import { Platform } from "react-native"
+import env from "react-native-config"
+
+export interface Configuration {
+  displaySelfAssessment: boolean
+  displayReportAnIssue: boolean
+  appDownloadLink: string
+  authorityPrivacyPolicyUrl: string
+  authorityName: string
+  authorityAdviceUrl: string
+}
+
+const initialState = {
+  displaySelfAssessment: false,
+  displayReportAnIssue: false,
+  appDownloadLink: "",
+  authorityPrivacyPolicyUrl: "",
+  authorityName: "",
+  authorityAdviceUrl: "",
+}
+
+const ConfigurationContext = createContext<Configuration>(initialState)
+
+const DEFAULT_DOWNLOAD_LINK = ""
+const DEFAULT_AUTHORITY_PRIVACY_URL = ""
+const DEFAULT_AUTHORITY_NAME = ""
+const DEFAULT_AUTHORITY_ADVICE_URL = ""
+
+const ConfigurationProvider: FunctionComponent = ({ children }) => {
+  const authorityPrivacyPolicyUrl =
+    env.PRIVACY_POLICY_URL || DEFAULT_AUTHORITY_PRIVACY_URL
+  const authorityName = env.GAEN_AUTHORITY_NAME || DEFAULT_AUTHORITY_NAME
```

Comment:
what do you think about be less liberal with the defaults? if we fail to provide an authority name, would we rather the app fail to build than to accidentally ship a `PathCheck`?

Reply:
HMW fail here? When? Is this a case of yeah this builds and then when I press play we show an error? Do we have any tests running after an environment is added?

Reply:
What do you think about a `bin/config_is_valid.sh` script that is run before builds on ci fastlane?  If its fast we might also want to add it to `yarn validate`

Reply:
so not a perfect solution but, my main concern is we are setting ourselves up to forget to update a config somewhere and ship a build that runs but shows the wrong app name or the like, which would not be a great look.

Reply:
100% agree with you on this one, my only concern is where to do this. Since it would only make sense to validate prior-creating an android or ios build. The binaries we have right now for scripting are made to be run on demand when someone is about to cut a build but we don't necessarily need those before each build. Also, those live far from the logic of the application.

Reply:
Should we do it on the place we store the configs?

Reply:
that could make sense as well.  dont have a strong opinion, but leaning towards keeping all the ci in one spot.